### PR TITLE
add swtichy.js v0.1.1 with autoupdate config

### LINKED ISSUE #7277: [Request] Add switchy
https://github.com/lou/switchy

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #7277 
@dannyxx001 please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/lou/switchy
 * Official website (optional, not the repository): http://lou.github.com/switchy/
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 72
   - Count of watchers: 7
   - Count of forks: 16
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7277, cc @lou

### CODE REVIEW

File: ajax/libs/switchy.js/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "dependencies": {
+    "jquery": ">= 1.8"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/lou/switchy.git",
+    "basePath": "",
+    "files": [
+      "*.js",
+      "*.css"
```
Comment (PeterDaveHello) [inline ajax/libs/switchy.js/package.json]: 1. Use `"*.+(css|js|map)"` is better then `"*.js",` + "*.css"`
2. Should not use this kind of wildcard in non-dist folder/repo, try `"switchy*.+(css|js|map)"`

**Comment by PeterDaveHello** [comment]
@pvnr0082t You should tag me to review your PR, not to @cdnjs/intern2 until you finished the first training.

File: ajax/libs/switchy.js/package.json (hunk 1 above)
Comment (pvnr0082t) [inline ajax/libs/switchy.js/package.json]: OK. I'll try it. Thanks.

**Comment by pvnr0082t** [comment]
@PeterDaveHello  Okay. I got you. Thank you.

**Comment by pvnr0082t** [comment]
@PeterDaveHello I have already updated ```package.json``` by using  ```"switchy*.+(css|js|map)``` in autoupdate-config.
Please help me review this pull request again. Thank you.

**Comment by PeterDaveHello** [comment]
@pvnr0082t you mixed unrelated commit here, please double check it.

**Comment by pvnr0082t** [comment]
@PeterDaveHello I fixed the problem mixed unrelated commit. Thanks.

**Comment by PeterDaveHello** [comment]
@pvnr0082t you should rebase your branch with latest master branch.

**Comment by pvnr0082t** [comment]
@PeterDaveHello I rebased the branch with latest master branch.

**Comment by PeterDaveHello** [comment]
Thanks for your contribution.

**Pull request merged**